apps/meteor/app/slashcommands-invite/server/server.ts

```diff
@@ -11,74 +10,75 @@ import { api } from '../../../server/sdk/api';
  * Invite is a named function that will replace /invite commands
  * @param {Object} message - The message object
  */
-
-function Invite(_command: 'invite', params: string, item: IMessage): void {
-	const usernames = params
-		.split(/[\s,]/)
-		.map((username) => username.replace(/(^@)|( @)/, ''))
-		.filter((a) => a !== '');
-	if (usernames.length === 0) {
-		return;
-	}
-	const users = Meteor.users.find({
-		username: {
-			$in: usernames,
-		},
-	});
-	const userId = Meteor.userId() as string;
-	if (users.count() === 0) {
-		api.broadcast('notify.ephemeralMessage', userId, item.rid, {
-			msg: TAPi18n.__('User_doesnt_exist', {
-				postProcess: 'sprintf',
-				sprintf: [usernames.join(' @')],
-				lng: settings.get('Language') || 'en',
-			}),
-		});
-		return;
-	}
-	const usersFiltered = users.fetch().filter(function (user) {
-		const subscription = Subscriptions.findOneByRoomIdAndUserId(item.rid, user._id, {
-			fields: { _id: 1 },
-		});
-		if (subscription == null) {
-			return true;
+slashCommands.add(
+	'invite',
+	(_command: 'invite', params, item): void => {
```

Comment:
Nesting this function declaration as a parameter expression damages readability, as evidenced by the indentation increase.